racker/salus-telemetry-api
# Adam9500/salus 142 spqr admin api

# Resolves

    salus-142

# What

    Convert from graphql to graphql SPQR

# How

    Switches the libraries that we are using. Removes the configs that are no longer necessary and adds proper annotations 

## How to test

    Run GraphQL queries against the endpoints to verify that they are working correctly.. They are currently not.

# Why

    GraphQL SPQR is more of a code first approach which makes it much easier to work with the code. Much less duplication because it allows the library to imply the configurations based off of the code and annotations we give it.

# TODO

    The mutations are giving validation errors for unknown reasons
